tools/confix/go.mod

```diff
@@ -154,7 +154,11 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-// Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
-// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
-// TODO investigate if we can outright delete this dependency, otherwise go install won't work :(
-replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
+replace (
+	// TODO: remove replacement
+	github.com/cosmos/cosmos-sdk => ../../../cosmos-sdk
```

Comment:
how do you handle this? It would have to point to my fork of the SDK to include the made changes?

Reply:
We can handle that in a follow up once merged.